contribute/release-policy.md

```diff
@@ -0,0 +1,84 @@
+---
+id: release-policy
+title: Release policy
+---
+
+## Release semantics
+
+The Pulsar project follows a variant of [Semantic Versioning](http://semver.org/spec/v2.0.0.html). Existing releases can expect patches for bugs and security vulnerabilities. New features will target minor releases. The difference is that a major version bump will not carry any special meaning in terms of "big features" included in the release or breaking API changes. Instead, it would simply signal a new long-term support (LTS) release.
+
+For example,
+
+* 2.10.0 is a feature release;
+* 2.10.1 is a patch release;
+* 2.11.0 is a feature release;
+* 3.0.0 is the first LTS release;
+* 3.1.0 is a feature release;
+* 3.2.0 is a feature release;
+* 3.2.1 is a patch release;
```

Comment:
Change to "major/minor" to keep consistent? Can you check all the occurencies in the whole doc?

Reply:
Please take a look at:

* https://github.com/apache/pulsar-site/pull/433#discussion_r1115172798
* https://github.com/apache/pulsar-site/pull/433#issuecomment-1441130956

I can update all the occurrences but we need some consensus first.

Reply:
We should also add 

`3.0.1 is the a patch release of the LTS version`

Reply:
SGTM.